dapps/marketplace/src/pages/transactions/Purchases.js

```diff
@@ -126,40 +126,44 @@ const NoPurchases = () => (
 )
 
 const Purchase = ({ listing, offer }) => (
-  <div className="purchase">
-    <div className="pic">
-      <Pic listing={listing} />
-      {offer.quantity === undefined || offer.quantity <= 1 ? null : (
-        <div className="quantity">{offer.quantity}</div>
-      )}
-    </div>
-    <div className="details">
-      <div className="top">
-        <Link className="title mb-1" to={`/purchases/${offer.id}`}>
-          {listing.title || <i>Untitled Listing</i>}
-        </Link>
-        <div className="right">
-          <span className="time-estimate">
-            {distanceToNow(offer.createdEvent.timestamp, true) + ' ago'}
-          </span>
-          <OfferStatus offer={offer} />
+  <Fragment>
```

Comment:
Any reason to wrap the whole `<Link>` element in a `<Fragment>`?

Reply:
You're right! Somehow I got the impression that a Link wouldn't serve as a wrapper to divs, that's why I included the Fragment.

Already fixed it, just running the tests to push to remote.